test/integration/basic_test.go

```diff
@@ -154,7 +171,7 @@ func FeatureContext(s *godog.Suite) {
 
 	s.BeforeScenario(func(interface{}) {
 		testDir := setUp()
-		os.RemoveAll(testDir)
+		fmt.Println("Running Integration test in : ", testDir)
```

Comment:
This will help to find where the actual test is running to findout which directory it's in.